docs/user-docs/latest/settings/managing-global-settings.md

```diff
@@ -55,6 +55,25 @@ To add a new locale:
 4. (optional) In the Advanced settings tab, tick the *Set as default locale* setting to make your new locale the default one for your Strapi application.
 5. Click on the **Save** button to confirm the addition of your new locale.
 
+## Configuring the Media Library
+
+The [Media Library](/user-docs/latest/media-library/introduction-to-media-library.md) displays all assets uploaded in the Strapi application. The Media Library settings allow  controlling the format, flie size, and orientation of uploaded assets.
```

Comment:
Line 60, you'll find 2 typos that were fixed in another commit 😅 
👉 https://github.com/strapi/documentation/pull/867/commits/a2111c3b08a78fd47b931471aaffcfe0298c0d66